.script/tests/detectionTemplateSchemaValidation/DetectionsYamlFilesTestData.cs

```diff
@@ -11,20 +11,36 @@ public class DetectionsYamlFilesTestData : TheoryData<string>
     {
         public DetectionsYamlFilesTestData()
         {
-            string detectionPath = GetDetectionPath();
-            var files = Directory.GetFiles(detectionPath, "*.yaml", SearchOption.AllDirectories).ToList();
+            List<string> detectionPath = GetDetectionPath();
+            var files = Directory.GetFiles(detectionPath[0], "*.yaml", SearchOption.AllDirectories).ToList();
+            files.AddRange(Directory.GetFiles(detectionPath[1], "*.yaml", SearchOption.AllDirectories).ToList().Where(s => s.Contains("Analytic Rules")));
```

Comment:
this filtering should be done in GetDetectionPaths method.

Reply:
Some .yaml file names present in hunting queries as well with same name, hence applying filter in solution folder with "Analytic Rules".